formulas/wick-base/explorers/machine-type

```diff
@@ -29,5 +29,11 @@ for I in {{1..3}}; do
     fi
 done
 
+# Try to detect if we are in a docker container
+if cut -d'/' -f2 /proc/self/cgroup | grep '^docker$' 2>&1 > /dev/null; then
```

Comment:
Would prefer to not quote unless necessary and also to use `-q` with `grep`.  Also, one should see if the file exists.

    if [[ -f /proc/self/cgroup ]] && cut -d / -f 2 /proc/self/cgroup | grep -q '^docker$'; then